bigframes/session/__init__.py

```diff
@@ -504,11 +505,12 @@ def read_gbq_table(
             api_name="read_gbq_table",
         )
 
-    def _read_gbq_table_to_ibis_with_total_ordering(
+    def _get_snapshot_sql_and_primary_key(
         self,
         table_ref: bigquery.table.TableReference,
         *,
         api_name: str,
+        enforce_region: bool = False,
```

Comment:
Could you add some docstring for this? It's not 100% clear to me what this means just from the variable name.